build.sbt

```diff
@@ -142,6 +142,7 @@ lazy val `OPAL` = (project in file("."))
   .settings((Defaults.coreDefaultSettings ++ Seq(publishArtifact := false)): _*)
   .enablePlugins(ScalaUnidocPlugin)
   .settings(
+    libraryDependencies += Dependencies.scalaCompiler(scalaVersion.value),
```

Comment:
Wouldn't it make sense to define this in the Dependencies file like the values for, e.g., bi and br?